tasks/mutations/personal_task_template_mutations.ts

```diff
@@ -10,36 +10,54 @@ export type TaskTemplateDTO = {
   isPublicVisible: boolean
 }
 
-const queryKey = 'taskTemplates'
+type QueryKey = 'personalTaskTemplates'| 'wardTaskTemplates'
```

Comment:
Constant definitions would be useful...
`const personalTaskTemplates = "personalTaskTemplates"` ?

Reply:
but then you can't really use them as a parameter or do you mean something like: `const queryKey = ['personalTaskTemplate', 'wardTaskTemplate]`